mods/gen4/formats-data.js

```diff
@@ -1990,12 +1990,12 @@ exports.BattleFormatsData = {
 	},
 	mesprit: {
 		inherit: true,
-		randomBattleMoves: ["hiddenpowerfire", "thunderbolt", "icebeam", "psychic", "thunderwave", "uturn", "stealthrock", "trick"],
+		randomBattleMoves: ["hiddenpowerfire", "thunderbolt", "icebeam", "psychic", "calmmind", "substitute", "thunderwave", "uturn", "stealthrock", "trick"],
```

Comment:
I would remove Trick if you want to add this. Seems like potential for bad combos.